ota-plus-web/app/reactapp/src/stores/data/newFeatures.js.new.js

```diff
@@ -0,0 +1,380 @@
+/** @format */
```

Comment:
This file is not used in any place. Please remove it.

Reply:
oops, sorry not sure how that got in there - fixed